src/profile-logic/comparison.js

```diff
@@ -706,7 +706,9 @@ function combineSamplesDiffing(
         `);
       }
       newSamples.stack.push(newStackIndex);
-      newSamples.responsiveness.push(samples1.responsiveness[i]);
+      // Diffing event delay values doesn't make sense since interleaved values
+      // of eventDelay/responsiveness don't mean anything.
+      newSamples.eventDelay.push(null);
```

Comment:
We talked about this with @julienw and he is in favour of pushing null here instead of responsiveness values, since interleaved responsiveness values don't mean anything.